.github/workflows/build.yml

```
@@ -50,6 +51,9 @@ jobs:
           - suite: rails-6.0
             spec: --tag probe:rails --tag ~probe
             env: BUNDLE_GEMFILE=gemfiles/rails_60.gemfile
+          - suite: redis-4
```

This one needs `suite: redis-4.0` to match the `suite` value from the list above.

```suggestion
          - suite: redis-4
          - suite: redis-4.0
```

---

Nice catch, thanks. Fixed.